universal-login-commons/src/core/utils/ValuePresenter.ts

```diff
@@ -0,0 +1,12 @@
+import {isNumber} from './isNumber';
+
+export class ValuePresenter {
+  static presentRoundedValue(value: string, roundFunc: (value: string, precision?: number) => string, precision?: number): string {
+    if (value === '') {
+      return '';
+    } else if (!isNumber(value)) {
```

Comment:
why not just throw an error if the value is not a number or is `''` ?

Reply:
I think it should stay like this, but we can discuss it